lib/mayaUsd/utils/util.cpp

```diff
@@ -1937,6 +1937,10 @@ std::pair<bool, std::string> UsdMayaUtil::ValueToArgument(const VtValue& value)
 {
     if (value.IsHolding<bool>()) {
         return std::make_pair(true, std::string(value.Get<bool>() ? "1" : "0"));
+    } else if (value.IsHolding<int>()) {
+        return std::make_pair(true, std::to_string(value.Get<int>()));
+    } else if (value.IsHolding<float>()) {
+        return std::make_pair(true, std::to_string(value.Get<float>()));
```

Comment:
We have int and float values in the args: startTime, endTime, frameStride, so we need a way to unparse them.